### PR TITLE
Add Content for Substrate Files

### LINKED ISSUE #3042: Add Context for Files Module on PolkadotJS
image

image

### PR DESCRIPTION
**Pull request by salmad3**
resolves #3042 

originally included more descriptive content, but now links to the key components and uses screenshots from the PR that was opened to include the file module on PolkadotJS, which are clean

cc. @badkk

### CODE REVIEW

File: docs/build/build-storage.md [hunk 1]
```diff
@@ -98,3 +98,40 @@ Try integrating with Crust by following their [Crust Storage 101](https://wiki.c
...
+You should be able to view the file info, as follows:
+
+![file info](../assets/files/substrate-files-4.png)
+
+As well as the file status:
+
+![file statue](../assets/files/substrate-files-5.png)
+
+The whole files module is decentralized, so your file directory is cached in browser. 
+You can also export and import your file directory info.
```
Comment (profradha) [inline docs/build/build-storage.md]: The reason why export/import needs to be done should be highlighted to the user

```suggestion
:::note

The whole files module is decentralized, so your file directory is only cached in browser. 
The file directory info will not be visible if you switch to a new browser or clear the browser cache. 
The storage module allows you to export file directory info from the current browser and import it to the new browser.

:::
```

File: docs/build/build-storage.md [hunk 2]
```diff
@@ -98,3 +98,40 @@ Try integrating with Crust by following their [Crust Storage 101](https://wiki.c
...
+
+![manage file dir](../assets/files/substrate-files-5.png)
```
Comment (profradha) [inline docs/build/build-storage.md]:
```suggestion
![manage file dir](../assets/files/substrate-files-6.png)
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Providing context on why export/import exists

**Review by DrW3RK — APPROVED** [review]
good to merge!

**Pull request merged**